search-relevancy-test/src/search_relevancy_test/reporter.clj

```diff
@@ -20,6 +20,40 @@
                    (ansi/style position :red)))
                index-order))
 
+(defn- result-discounted-cumulative-gain
+ "Calculate the discounted cumulative gain for the results.
+ The formula for DCG is: for each position i (starting at 1), calculate:
+ (position relevancy / log2(i + 1))
+ 'position relevancy' is how relevant the result was, so higher is better. When
+ the position comes back lower is better so take the inverse of the position to use
+ in the calculations"
```

Comment:
I think it might be more clear to say:
'position relevancy' is calculated using the inverse of the position because the lower the position the result came back at, the more relevant the result was.